docs/preview/03-Features/powershell/azure-devops.md

```diff
@@ -120,6 +120,92 @@ In Azure DevOps, below permissions need to be set on your variable group in orde
+output ApplicationInsights_ConnectionString string = reference(applicationInsight.id, '2020-02-02').ConnectionString
+```
+
+This Bicep template will deploy the Application Insights instance and place the `Id` and `ConnectionString` in the output parameters. 
+
+##### Updating The Variable Group
+Now all we need to do is execute our [script](#setting-arm-outputs-to-azure-devops-variable-group) which will update the Azure DevOps variable group.
+
+From an Azure DevOps pipeline this can be done like so:
+``` powershell
+Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+Install-Module -Name Arcus.Scripting.DevOps -AllowClobber
+
+Set-AzDevOpsArmOutputsToVariableGroup -VariableGroupName 'myVariableGroup'
+```
+
+##### Combining It All In A Pipeline
+Now that we have walked through both steps, let's take a look on how to combine all this into an Azure DevOps pipeline.
+For this we use YAML and define two tasks, the first will deploy our Application Insights instance and the second will update our Azure DevOps variable group.
+
+``` yaml
+- task: AzureResourceGroupDeployment@3
+  displayName: 'Deploy Bicep template'
+  inputs:
+    azureResourceManagerConnection: 'myServiceConnection'
+    subscriptionId: 'mySubscriptionId'
+    resourceGroupName: 'myResourceGroup'
+    location: 'West Europe'
+    csmFile: 'applicationInsights.bicep'
+    csmParametersFile: 'applicationInsights.parameters.json'
+    deploymentOutputs: ArmOutputs
+
+- task: PowerShell@2
+  displayName: 'Update Variable Group'
+  env:
+    system_accesstoken: $(System.AccessToken)
+    ArmOutputs: $(ArmOutputs) # only needs to be set for Linux agents
+  inputs:
+    targetType: 'inline'
+    script: |
+      Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+      Install-Module -Name Arcus.Scripting.DevOps -AllowClobber
+
+      Set-AzDevOpsArmOutputsToVariableGroup -VariableGroupName 'myVariableGroup' -ArmOutputsEnvironmentVariableName 'ArmOutputs' -UpdateVariablesForCurrentJob
+```
+
+There are a few things worth noting. First of all we define `deploymentOutputs: ArmOutputs` during the `AzureResourceGroupDeployment@3` task. This means that the output parameters we specified in our Bicep template will be placed in a variable called `ArmOutputs`, this is then referenced during the execution of our script with `-ArmOutputsEnvironmentVariableName 'ArmOutputs'`.
+
+Secondly we use `-UpdateVariablesForCurrentJob` as a parameter when calling the script. This means that the output parameters from the Bicep file are also available as pipeline variables in the current running job. While not necessary in our example here, if you need to deploy another Bicep template that needs output parameters from an earlier deployed Bicep template this is the way to do it.
+
+Finally we use `system_accesstoken: $(System.AccessToken)` in the `Powershell@2` task, this is necessary because we need to use the security token used by the running build. 
+
```

Comment:
I would add an explicit remark here that you also need to reference the ArmOutputs variable in your environment-variables section. (see line 185).

Something like this:

```suggestion
> Please note that the `ArmOutputs` variable is not available 'as is' when executing the Powershell task on a Linux agent.  When using a Linux agent, you have to explicitly add that variable in the `env:` section of the Powershell task.
```